posix.mak

```diff
@@ -738,7 +738,7 @@ $G/docs-latest.json : ${DMD_LATEST} ${DMD_LATEST_DIR} \
 		sed -e /mscoff/d $${DMD_EXCLUDE_LATEST_BASH} ${DMD_EXCLUDE_LATEST}
 	find ${DRUNTIME_LATEST_DIR}/src -name '*.d' | \
 		sed -e /unittest.d/d -e /gcstub/d >> $G/.latest-files.txt
-	find ${PHOBOS_LATEST_DIR} -name '*.d' | \
+	find ${PHOBOS_LATEST_DIR} -name '*.d' | grep -v generated | \
```

Comment:
This doesn't look right. `PHOBOS_LATEST_DIR=$G/phobos-${LATEST}` and `G=$(GENERATED)` and `GENERATED=.generated` so this will filter out everything, won't it?